tests/robustness/powerfailure_test.go

```diff
@@ -0,0 +1,197 @@
+//go:build linux
+
+package robustness
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"go.etcd.io/bbolt/tests/dmflakey"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sys/unix"
+)
+
+// TestRestartFromPowerFailure is to test data after unexpected power failure.
+func TestRestartFromPowerFailure(t *testing.T) {
+	flakey := initFlakeyDevice(t, t.Name(), dmflakey.FSTypeEXT4, "")
+	root := flakey.RootFS()
+
+	dbPath := filepath.Join(root, "boltdb")
+
+	args := []string{"bbolt", "bench",
+		"-work", // keep the database
+		"-path", dbPath,
+		"-count=1000000000",
+		"-batch-size=5", // separate total count into multiple truncation
+	}
+
+	logPath := filepath.Join(t.TempDir(), fmt.Sprintf("%s.log", t.Name()))
+	logFd, err := os.Create(logPath)
+	require.NoError(t, err)
+	defer logFd.Close()
+
+	// FIXME: gofail should support unix socket so that the test cases won't
+	// be conflicted.
```

Comment:
Where is the conflict? 
- The existing https://github.com/etcd-io/bbolt/blob/master/tests/failpoint/db_failpoint_test.go doesn't require exporting a port for failpoint;
- We will not run the test cases under test/robustness in parallel.

Reply:
I run into port-already-being-used issue in etcd flakey case when that test case doesn't close port after finish.
I remove that comment since it seems confusing.